package.json

```diff
@@ -94,7 +94,7 @@
     "tsup": "^7.2.0",
     "typescript": "^5.0.4",
     "vite": "^4.2.1",
-    "vitest": "^0.30.1",
+    "vitest": "^2",
```

Comment:
can you change this to 2.0.0

Reply:
Done. Also updated the deprecated `@vitest/coverage-c8` to a newer `@vitest/coverage-v8`, since the `vitest` is now version 2.